language/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -34,19 +42,24 @@ pub struct ProcessedModule {
 
 pub struct CompiledState<'a> {
```

Comment:
Now with more and more stuff being put into this struct, the name "CompiledState" seems pretty inaccurate. How about something like "AdapterState"?

Reply:
Maybe... It is still only really compiler information in way. Maybe more like `MoveProgramState` but thats confusing because its state for the program text, not the program executing